### PR TITLE
[api][web][SIMS-#227] Fix API and UI table display issue

### PR DESCRIPTION
**Pull request by fwpushan**
----
1. Fix api to accept multiple location
2. Fix UI table layout issue
--

### CODE REVIEW

File: sources/packages/api/src/route-controllers/institution/institution.controller.ts [hunk 1]
```diff
@@ -27,6 +27,7 @@ import { InstitutionUserRespDto } from "./models/institution.user.res.dto";
 import { InstitutionUserAuthDto } from "./models/institution-user-auth.dto";
 import { InstitutionUserRole, InstitutionUserType } from "../../types";
 import { InstitutionUserTypeAndRoleResponseDto } from "./models/institution-user-type-role.res.dto";
+import { User } from "src/database/entities";
```
Comment (andrewsignori-aot) [inline sources/packages/api/src/route-controllers/institution/institution.controller.ts:30]: Please change to a relative path.

File: sources/packages/api/src/route-controllers/institution/institution.controller.ts [hunk 2]
```diff
@@ -138,22 +139,35 @@ export class InstitutionController extends BaseController {
...
+      userEntity = await this.userService.getUser(`${body.userGuid}@bceid`);
     }
 
-    // Create User
-    const userEntity = this.userService.create();
-    userEntity.email = accountDetails.user.email;
-    userEntity.firstName = accountDetails.user.firstname;
-    userEntity.lastName = accountDetails.user.surname;
-    userEntity.userName = `${accountDetails.user.guid}@bceid`;
+    if (!userEntity) {
```
Comment (andrewsignori-aot) [inline sources/packages/api/src/route-controllers/institution/institution.controller.ts:148]: Can you please elaborate in which scenario the userEntity will be null at this stage?

File: sources/packages/api/src/route-controllers/institution/institution.controller.ts [hunk 3]
```diff
@@ -138,22 +139,35 @@ export class InstitutionController extends BaseController {
...
+      // Get user details
+      const accountDetails = await this.accountService.getAccountDetails(
```
Comment (andrewsignori-aot) [inline sources/packages/api/src/route-controllers/institution/institution.controller.ts:150]: This method is supposed to be called all the time, right?
Why is it being called only if "userEntity" is not defined?

**Review by andrewsignori-aot — COMMENTED** [review]
Please see the comments.

File: sources/packages/api/src/route-controllers/institution/institution.controller.ts (hunk 1 above)
Comment (fwpushan) [reply sources/packages/api/src/route-controllers/institution/institution.controller.ts]: Fixed

File: sources/packages/api/src/route-controllers/institution/institution.controller.ts (hunk 3 above)
Comment (fwpushan) [reply sources/packages/api/src/route-controllers/institution/institution.controller.ts:152]: Please check code comments.

Comment (andrewsignori-aot) [reply sources/packages/api/src/route-controllers/institution/institution.controller.ts:152]: Still not clear to me. We should always check for "Get user details from BCeID". Why is it inside an "if (!userEntity)"?

Comment (fwpushan) [reply sources/packages/api/src/route-controllers/institution/institution.controller.ts:152]: BCeID service call is require to get user info... if same user already exits we skip that call and create auth association.

Comment (andrewsignori-aot) [reply sources/packages/api/src/route-controllers/institution/institution.controller.ts:152]: How do we ensure that the user belongs to the institution that it is being associated with?

**Review by abschwenker — APPROVED** [review]
Approved to merge due to team changes. Will note comments for review of outstanding comments

File: sources/packages/web/src/views/institution/InstitutionUserDetails.vue [hunk 4]
```diff
@@ -131,7 +139,7 @@ export default {
     const invalidUserType = ref(false);
     const users = ref([] as InstitutionUserViewModel[]);
```
Comment (ann-aot) [inline sources/packages/web/src/views/institution/InstitutionUserDetails.vue:140]: I guess, we missed the update `users` after a new user is added

**Pull request merged**